src/aspire/utils/misc.py

```diff
@@ -109,3 +111,102 @@ def sha256sum(filename):
             h.update(mv[:n])
 
     return h.hexdigest()
+
+
+def gaussian_2d(resolution, x0=0, y0=0, sigma_x=1, sigma_y=1, peak=1, dtype=np.float64):
+    """
+    Returns a 2d Gaussian in a square 2d numpy array.
+
+    Default is a centered disc of spread=peak=1.
+
+    Note for odd resolutions center is the grid point 0,
```

Comment:
Isn't the center at zero for all sizes?

Reply:
is zero a grid point or a pixel... I'll convert to grid_2d per your other comment so we have the same convention.  I didn't think it was terribly important for the ut helper functions, but you are right, they should all be consistent.

Reply:
:pray: